strings_test.go

```diff
@@ -29,6 +29,14 @@ func TestTrunc(t *testing.T) {
 	if err := runt(tpl, "foo"); err != nil {
 		t.Error(err)
 	}
+	tpl = `{{ "baaaaaar" | trunc -3 }}`
```

Comment:
Consider adding a test case for `trunc -999` to make sure it doesn't blow up if out of bounds

Reply:
Sure.

I tested this locally but didn't include a test case. Adding now.

Reply:
`s trunc -999` will result in `s` being returned. This behaviour is unchanged with this PR. - Just so you know.